orion-ui/package.json

```diff
@@ -32,7 +32,7 @@
     "dotenv": "16.0.2",
     "eslint": "^8.23.1",
     "ts-node": "10.9.1",
-    "typescript": "^4.8.2",
+    "typescript": "^4.8.3",
```

Comment:
Did you mean to update typescript in this PR?

Reply:
Hmmm.. nope. Let me update that.